app/components/TrackComponent/index.js

```diff
@@ -0,0 +1,78 @@
+/**
+ *
+ * TrackComponent
+ *
+ */
+
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { isEmpty } from 'lodash';
+import { colors } from '@app/themes/index';
+import If from '@components/If';
+import { Card, Image, Typography, Button } from 'antd';
+
+const { Title, Paragraph } = Typography;
+
+const TrackCardContainer = styled(Card)`
+  && {
+    border-radius: 0.5rem;
+    width: ${props => (props.width ? props.width : '25rem')};
+    border: 1px solid ${colors.secondary};
+    text-align: center;
+`;
+
+const StyledDescription = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const PlayTrackBtn = styled(Button)`
+  && {
+    background-color: ${colors.btnBackground};
+    border: none;
+    color: ${colors.btnText};
+    border-radius: 0.5rem;
+    width: 100%;
+  }
+`;
+
+export function TrackComponent({ collectionName, artistName, imgUrl, trackName }) {
+  return (
+    <TrackCardContainer data-testid="track-component">
+      <If condition={!isEmpty(imgUrl)} otherwise={<Image>No image available</Image>}>
+        <Image src={imgUrl} width="80%" height="250px" />
```

Comment:
Do we need to use px ? Otherwise keep it in rem

Reply:
Removed the height attribute